src/quo2/components/avatars/user_avatar.cljs

```diff
@@ -134,7 +134,7 @@
       :style               {:width         outer-dimensions
                             :height        outer-dimensions
                             :border-radius outer-dimensions}}
-     (when (and ring? identicon?)
+     (when (and false (and ring? identicon?)) ;;TODO not implemented yet
```

Comment:
can we add a corresponding issue for this?

Reply:
i believe we have it

Reply:
let's include the link here so :)